src/gen/conditions.rs

```diff
@@ -2681,7 +2664,7 @@ fn test_multiple_create_coin() {
         amount: 43_u64,
         hint: a.null()
     }));
-    assert_eq!(spend.flags, ELIGIBLE_FOR_DEDUP);
+    assert_eq!(spend.flags, ELIGIBLE_FOR_DEDUP | ELIGIBLE_FOR_FF);
```

Comment:
this test exercises the case where there's an output with a different amount, but still odd. This expected result changed because of the logic change above